### ISSUE TITLE
preserve_newlines don't work for js in html if max_preserve_newlines set

### ISSUE THREAD

**Issue by s-trooper** [beautify-web/js-beautify#915]
Hi, i have this config but the new lines are not preserved for JS in html like this:

``` js
{
    "max_preserve_newlines": 10,
    "preserve_newlines": true
}
```

but in this case, without max_preserve_newlines set, new lines are preserved !

``` js
{
    "preserve_newlines": true
}
```

example file:

``` html
<body>
    <div></div>
  <script>
  var x = "1";

  var y = "2";
  </script>
</body>
```

**Comment by bitwiseman** [comment]
Have you tried this on the website?

**Comment by s-trooper** [comment]
i've just tested with website version.
```"Allow 1 newline between tokens"``` option will don't keep new line in JS but HTML. (its seems i was using  "max_preserve_newlines": 1 not 10 by my npm/pip tests)
```"Allow 2 newline between tokens"``` will keep 1 line in JS and 2 in HTML.

Thank you for the hint, now i see the mechanics and its OK but for me it's still minor bug.

**Comment by aszx0413** [comment]
is this issue still not fixed?
when a HTML file with javascript inside, like
```
<div>xxx</div>
<script>
var a = 1;
var b = 2;
</script>
```
`"max_preserve_newlines": 2` will keep 2 blank lines in HTML code block, but keep 1 blank line in Javascript code.
Confuse.

**Comment by bitwiseman** [comment]
@aszx0413 
No, the behavior is unchanged.  Not sure why we closed it except that it is an old issue with no work done on it. 

The difference is likely that the javascript beautifier by someone thinking about preserving at most 2 newlines, while the html beautifier was implemented by someone thinking of preserving empty lines.  
 
It is a mismatch, that should be fixed, but part of that fix likely the name change to `preserve_empty_lines` as noted in #1391 .  

Is this an issue you care about enough to work on?
